We need to have the defines available before
we try to evaluate them



--- konqueror-embedded-snapshot-20030705/konq-embed/src/popuphandler.cc~	2002-12-05 13:47:07.000000000 +0100
+++ konqueror-embedded-snapshot-20030705/konq-embed/src/popuphandler.cc	2004-08-22 17:10:44.000000000 +0200
@@ -1,3 +1,6 @@
+
+#include <qnamespace.h>
+
 #include "popuphandler.h"
 
 #if defined(_WS_QWS_) && defined(QT_NO_QWS_CURSOR)
